### PR TITLE
Merge release 2.9 into trunk

### PR DESCRIPTION
**Pull request by mokagio**
Doing this earlier to re-enable the signup flow. Internal ref. paaHJt-1Sn-p2.

Leaving this as a draft till we actually release it. Still, I'd appreciate a review anyway.

Thanks 🙇‍♂️ .

### CODE REVIEW

File: Scripts/extract_release_notes.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# ruby ./this_script -k | pbcopy
+
+GITHUB_URL = 'https://github.com/Automattic/simplenote-macos'
+
+RELEASE_NOTES_FILE = 'RELEASE-NOTES.txt'
+NOTES = File.read(RELEASE_NOTES_FILE)
+lines = NOTES.lines
+
+def replace_pr_number_with_markdown_link(string)
+  string.gsub(/\#\d*$/) do |pr_number|
```
Comment (AliSoftware) [inline Scripts/extract_release_notes.rb:23]: Same feedback as in https://github.com/Automattic/simplenote-ios/pull/1202#pullrequestreview-608589162

**Review by AliSoftware — APPROVED** [review]
LGTM 👍 

Same wonder 🤔  than in https://github.com/Automattic/simplenote-macos/pull/912#pullrequestreview-608585964 about why `it/release_notes.txt` and `pt-BR/release_notes.txt` files were still around and not deleted in an earlier PR alongside other locales… but 🤷 

_PS: feedback about the `extract_release_notes` script is food for thoughts and not really related to the `release->trunk` of this PR, so feel free to just take note and address in a future, separate PR_

**Review by jleandroperez — APPROVED** [review]
:shipit: on my end sir!!

**Comment by mokagio** [comment]
This has now been published on the Store: Marking it "Ready for Review" and merging it.

**Pull request merged**